extra/3-magic-8-ball.js

```diff
@@ -59,9 +76,24 @@ function shakeBall() {
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
 function checkAnswer(answer) {
+  if (veryPositive.indexOf(answer) >= 0 ) {
+    console.log("very positive");
   //Write your code in here
+  } else if (positive.indexOf(answer) >= 0) {
+     console.log("positive");
+     } else if (negative.indexOf(answer) >= 0) {
+      console.log("negative");
+     } else {
+      console.log("very negative");
+     }
```

Comment:
```suggestion
 if (veryPositive.indexOf(answer) >= 0 ) {
    return "very positive";
  } else if (positive.indexOf(answer) >= 0) {
      return "positive";
     } else if (negative.indexOf(answer) >= 0) {
      return "negative";
     } else {
      return "very negative";
     }
```
console.log needs only for developer to see what's happening. As I see you just forgot to have "return" in your loop.